files/default/handler/audit_report.rb

```diff
@@ -124,7 +124,13 @@ def get_opts(format, quiet)
 
       # run profiles and return report
       def call(opts, profiles)
-        Chef::Log.info "Initialize InSpec #{::Inspec::VERSION}"
+        Chef::Log.info "Using InSpec #{::Inspec::VERSION}"
+
+        if Gem::Version.new(Inspec::VERSION) < Gem::Version.new('1.24.0')
```

Comment:
I'd store this in a constant somewhere near the top since the version string is used in two places: the comparison and the log message. That way we only have to change it in one location.